packages/xsnap-lockdown/scripts/build-bundle.js

```diff
@@ -14,29 +14,22 @@ const computeSha256 = bytes => {
   return hash.digest().toString('hex');
 };
 
-const make = async (name, entrypath, spec) => {
-  await fsp.mkdir(path.dirname(spec), { recursive: true });
+const make = async name => {
+  await fsp.mkdir(path.dirname(bundlePaths[name]), { recursive: true });
   const format = 'nestedEvaluate';
-  const bundle = await bundleSource(entrypath, { format });
+  const bundle = await bundleSource(entryPaths[name], { format });
   const bundleString = JSON.stringify(bundle);
   const sha256 = computeSha256(bundleString);
-  await fsp.writeFile(spec, bundleString);
-  return { length: bundleString.length, sha256, spec };
+  await fsp.writeFile(bundlePaths[name], bundleString);
+  await fsp.writeFile(hashPaths[name], `${sha256}\n`);
+  return { length: bundleString.length, sha256, where: bundlePaths[name] };
```

Comment:
Slight preference for keeping the bindings, possibly renamed.
```suggestion
const make = async name => {
  const spec = bundlePaths[name];
  const entryPath = entryPaths[name];
  const hashPath = hashPaths[name];
  await fsp.mkdir(path.dirname(spec), { recursive: true });
  const bundle = await bundleSource(entryPath, { format: 'nestedEvaluate' });
  const bundleString = JSON.stringify(bundle);
  const sha256 = computeSha256(bundleString);
  await fsp.writeFile(spec, bundleString);
  await fsp.writeFile(hashPath, `${sha256}\n`);
  return { length: bundleString.length, sha256, where: spec };
```